src/Migrations/Version20250521091607.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MatchBot\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Patch a donation to get Regression fund balances in sync.
+ *
+ * @see Version20250515172831 where I tried to do this but had it backwards. The donation here is a new
+ * £5 per donation mandate created to allow correcting regression fund ID 2.
+ */
+final class Version20250521091607 extends AbstractMigration
+{
+    private int $donationId = 306679;
+
+    public function getDescription(): string
+    {
+        return 'Patch a donation to £1 so balances are in sync, only in Regression';
+    }
+
+    public function up(Schema $schema): void
+    {
+        if (getenv('APP_ENV') !== 'regression') {
+            return;
+        }
+
+        $donationId = $this->donationId;
+        $this->addSql("UPDATE Donation SET amount = 1 WHERE id = $donationId LIMIT 1");
+        $this->addSql("UPDATE FundingWithdrawal SET amount = 1 WHERE donation_id = $donationId LIMIT 1");
```

Comment:
this can only run in regression so doesn't need very careful review - if we merge it and find the alarms there stop that's good.

I checked the donation and the FW in the db both currently have values of £5. I think it's only adjusting the FW that matters for the alarms we're getting but good to do both for consistency. Reducing the FW by £4 does look like it will bring things back into sync.

I think the fact that this is needed probably means the message output by the HandleOutOfSyncFunds could be clearer, but I don't think that's essential to do as part of this ticket.